src/app/common/TagList.vue

```diff
@@ -31,13 +39,18 @@ import type { RouteLocationNamedRaw } from 'vue-router'
 
 interface LabelValueWithRoute extends LabelValue {
   route: RouteLocationNamedRaw | undefined
+  isKuma: boolean
 }
 
 const props = withDefaults(defineProps<{
   tags: LabelValue[] | Record<string, string> | null | undefined
   shouldTruncate?: boolean
+  hideLabelKey?: boolean
+  alignment?: 'left' | 'right'
 }>(), {
   shouldTruncate: false,
+  hideLabelKey: false,
```

Comment:
How come we need this? How does someone know what the tag is if we only show the value? Is that what this is for?

Reply:
This is for the special case of the dataplane list where we show a specific subset of tags by value: service tags. The column reads "Services" so it’s unambiguous what they are. I wanted to avoid using a separate KTruncate/KBadge style listing for this.

Reply:
Oh I see! I quite like that, thanks!